src/main/webapp/app/exercises/programming/manage/instructions-editor/analysis/programming-exercise-instruction-analysis.service.ts

```diff
@@ -98,12 +98,10 @@ export class ProgrammingExerciseInstructionAnalysisService {
      *
      * @param issueType for which to retrieve the fitting translation.
      */
-    private getTranslationByIssueType = (issueType: ProblemStatementIssue) => {
+    private getTranslationByIssueType = (issueType: ProblemStatementIssue): string => {
```

Comment:
This """enum"" has only one possible value. If at any point a new value gets added, this method will throw a compilation error since then the missing default case will be a problem. That's exactly what I intended, force the developer to change this method in case a new issue type gets added.

Reply:
That makes sense. Could you add this under the return? Otherwise, in the future, a developer will just re-add it.